shared_model/interfaces/common_objects/string_types.hpp

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_SHARED_MODEL_STRING_TYPES_HPP
+#define IROHA_SHARED_MODEL_STRING_TYPES_HPP
+
+#include <string>
+
+#include <boost/serialization/strong_typedef.hpp>
+
+namespace shared_model {
+  namespace interface {
+    namespace types {
+      BOOST_STRONG_TYPEDEF(std::string, EvmCalleeHexString)
```

Comment:
It is not clear whether these types are usable outside tests or builders

Reply:
I think we can do without them now. I just wanted the seamlessly converted strong types as I described once.